cmd/kube-proxy/app/server.go

```diff
@@ -104,11 +112,15 @@ type Options struct {
 	master string
 	// healthzPort is the port to be used by the healthz server.
 	healthzPort int32
+
+	scheme *runtime.Scheme
```

Comment:
I could make it a package-level global but was trying to avoid that. The other thing I'll say is that `Options` is really just a struct that holds state/context for running this command, so something like `CommandContext` might be a more appropriate name.